### PR TITLE
DVISA-596: (feature) localization

### PR DESCRIPTION
**Pull request by fabioir-dedalus**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature: Localization for numbers and strings

* **What is the current behavior?** (You can also link to an open issue here)
German localization was applied to all numbers

* **What is the new behavior (if this is a feature change)?**
The library must get its localization initialized, then the localized numbers and strings will be applied

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Yes, the library needs to get its localization initialized

* **Other information**:

### CODE REVIEW

File: src/tools/annotation/AngleTool.js [hunk 1]
```diff
@@ -275,10 +275,9 @@ export default class AngleTool extends BaseAnnotationTool {
       const str = '00B0'; // Degrees symbol
 
       return (
-        toGermanNumberStringTemp(data.rAngle) +
+        localization.localizeNumber(data.rAngle) +
         String.fromCharCode(parseInt(str, 16)) +
         suffix
```
Comment (p1p3) [inline src/tools/annotation/AngleTool.js:280]: Should we localize the suffix of the angle tool?

File: src/tools/annotation/CobbAngleTool.js [hunk 2]
```diff
@@ -405,7 +405,7 @@ export default class CobbAngleTool extends BaseAnnotationTool {
 
     const suffix = !rowPixelSpacing || !colPixelSpacing ? ' (isotropic)' : '';
 
-    return `${toGermanNumberStringTemp(rAngle)}\u00B0${suffix}`;
+    return `${localization.localizeNumber(rAngle)}\u00B0${suffix}`;
```
Comment (p1p3) [inline src/tools/annotation/CobbAngleTool.js:408]: This is the same as the angle tool, should we consider "isotropic" in our localisation?

**Review by p1p3 — CHANGES_REQUESTED** [review]
The implementation was very quick and it looks good !

I might have one question : I saw that we have in multiple places 
```
    ? ` mm${String.fromCharCode(178)}`
     : ` px${String.fromCharCode(178)}`;
```

Should we move this logic into the localization.utils so it can be shared between the differente figures?

File: src/tools/annotation/AngleTool.js (hunk 1 above)
Comment (aminahasimbegovic) [reply src/tools/annotation/AngleTool.js:280]: A question for @LisaFrauenstein maybe?

File: src/util/localization/localization.utils.test.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      expect(localization.translate('average')).toBe('méd.');
+    });
+
+    it('should return a non recognized key', () => {
+      expect(localization.translate('thisKeyDoesNotExist')).toBe(
+        'thisKeyDoesNotExist'
+      );
+    });
+  });
+});
```
Comment (aminahasimbegovic) [inline src/util/localization/localization.utils.test.js:112]: Wow, awesome coverage! 👏

File: src/tools/annotation/CobbAngleTool.js (hunk 2 above)
Comment (aminahasimbegovic) [reply src/tools/annotation/CobbAngleTool.js:408]: @LisaFrauenstein ?

**Review by aminahasimbegovic — COMMENTED** [review]
Very nice work!

**Comment by fabioir-dedalus** [comment]
We agreed to apply the changes to a previous master version

**Pull request closed without merging**